packages/frontend/components/item/image-modal/component.tsx

```diff
@@ -0,0 +1,31 @@
+import { Modal, IconButton } from '@primitives';
+import { IconName, IconColor } from '@enums';
+import * as styles from './styles';
+
+interface ImageModalProps {
+  image: string;
+  isOpen: boolean;
+  setModalVisible: (value: boolean) => void;
+}
+
+export const ImageModal = ({
+  image,
+  isOpen,
+  setModalVisible,
+}: ImageModalProps) => (
+  <Modal visible={isOpen}>
+    <div css={styles.modalImageWrapper}>
+      <img src={image} alt="item" css={styles.modalImage} />
+      <div css={styles.modalClose}>
+        <IconButton
+          ariaLabel="closeModal"
+          icon={IconName.XMARK}
+          color={IconColor.ORANGE}
+          backgroundColor={'lightgray'}
+          size="md"
```

Comment:
maybe we need to add enum for this type of colors?

Reply:
we already have type 
![image](https://user-images.githubusercontent.com/71765510/188963743-232ce1c7-2166-45f8-961e-1c4c9562d939.png)